sc-memory/sc-memory/src/scs/scs_parser.cpp

```diff
@@ -479,23 +479,33 @@ void Parser::ProcessTriple(
 
     if (source.IsElementType() && (connector.GetType() == ScType::ConstPermPosArc))
     {
+      if (target.m_type.IsConnector())
+        SC_THROW_EXCEPTION(
+            utils::ExceptionParseError,
+            "Connector with type `" << std::string(target.m_type)
+                                    << "` can't belong to sc-element denoting type of sc-elements `" << source.GetIdtf()
+                                    << "`.");
+
```

Comment:
it should fail not because connector is added to some type(there are connector types like sc_membership_arc) but because ConstNodeNonRole cannot be extended to ConstNodeNonRoleClass

Reply:
common arc can be extended to membership arc?
![image](https://github.com/user-attachments/assets/80dc528c-8276-427c-9cd0-a3c16694decd)
![image](https://github.com/user-attachments/assets/00b0ba40-eac0-48f8-b1db-e7edf3ed82fc)